website/src/theme/BlogPostItem/index.js

```diff
@@ -0,0 +1,134 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
```

Comment:
This file is copied from Facebook?

Reply:
The file is a commit of the PR I recovered under https://github.com/apache/apisix-website/pull/616 and it appears to be a shell CLI generated file, https://docusaurus.io/docs/cli#docusaurus-swizzle-sitedir
cc @Baoyuantop have a look

Reply:
Yup, the blog page is not so customisable directly, so docusaurus has this functionality to `swizzle` out elements,
Refer to this - [`swizzle`](https://github.com/apache/apisix-website/issues/490#issuecomment-911850832)